components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth2/dao/TokenMgtDAO.java

```diff
@@ -1409,27 +1461,27 @@ public boolean validateScope(Connection connection, String accessToken, String r
         return false;
     }
 
-	/**
-	 * This method is used invalidate the existing token and generate a new toke within one DB transaction.
-	 *
-     * @param oldAccessTokenId     access token need to be updated.
-     * @param tokenState      token state before generating new token.
-	 * @param consumerKey     consumer key of the existing token
-	 * @param tokenStateId    new token state id to be updated
-	 * @param accessTokenDO   new access token details
-	 * @param userStoreDomain user store domain which is related to this consumer
-	 * @throws IdentityOAuth2Exception
-	 */
+    /**
+     * This method is used invalidate the existing token and generate a new toke within one DB transaction.
```

Comment:
Few typos in the method comment. Should be corrected as,
_This method is used **to** invalidate the existing token and generate a new toke**n** within one DB transaction._

Reply:
done with https://github.com/wso2-extensions/identity-inbound-auth-oauth/pull/472